packages/ensnode-utils/src/helpers.ts

```diff
@@ -0,0 +1,5 @@
+export const uniq = <T>(arr: T[]): T[] => [...new Set(arr)];
```

Comment:
this file can/should live in the indexer, it doesn't need to be shared. and the file name helpers really only makes sense in that context.